src/fonduer/utils/data_model_utils/structural.py

```diff
@@ -169,25 +169,26 @@ def get_ancestor_id_names(
     return id_names
 
 
-def common_ancestor(c: Tuple[SpanMention, SpanMention]) -> List[str]:
-    """Return the path to the root that is shared between a binary-Mention Candidate.
+def common_ancestor(c: Tuple[SpanMention, ...]) -> List[str]:
```

Comment:
Can we have a test about this with multinary mention candidate?

Reply:
Done. 
`tests/utils/data_model_utils/test_structural.py`